test/runner.test.js

```diff
@@ -18,6 +24,12 @@ describe('runner', function () {
     }
   }}));
 
+  it('looks for `package.json` using findup, starting with process.cwd()', function () {
```

Comment:
I've just realized... This spec dictates the implementation (`…using findup…`). Whether we use a direct `require` or `findup` to find `package.json` is an implementation detail IMO and doesn't add much, given we already have tests that cover executing the hooks specified in the `package.json`.

I'll put up a PR addressing this and a few other minor cleanups.